build-sensor/src/Main.cpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by Timothy Findlay on 2019-06-17.
+//
+
+#include <Arduino.h>
+#include "WiFi.h"
+#include <PubSubClient.h>
+#include "secrets.h"
+
+#include "step1.h"    // Step1 - Make the LED's blink
+#include "step2.h"    // Step2 - Add logging via Serial interface
+#include "step3.h"    // Step3 - Add WiFi connection
+#include "step4.h"    // Step4 - Add MQTT connection
+#include "step5.h"    // Step5 - Add Motion Sensor
+
+step5 o;
```

Comment:
probably would use a more descriptive name than just `o` for this.. 
```suggestion
step5 iotThing;
```

Reply:
Yes, was not very descriptive.